IDEAS/Examples/IBPSA/SingleZoneResidentialHydronic.mo

```diff
@@ -231,66 +249,34 @@ This is a single zone residential hydronic system model
 for WP 1.2 of IBPSA project 1. 
 <h3>Building Design and Use</h3>
 <h4>Architecture</h4>
-<p>
-This building envelope model corresponds to the BESTEST case 900 test case. 
-It consists of a single zone with a rectangular floor plan
-of 6 by 8 meters and a height of 2.7 m. 
-The zone further consists of two south-oriented windows of 6 m2 each, 
-which are modelled using a single window of 12 m2.
-</p>
+<p>This building envelope model corresponds to the BESTEST case 900 test case. It consists of a single zone with a rectangular floor plan of 6 by 8 meters and a height of 2.7 m. The zone further consists of two south-oriented windows of 6 m2 each, which are modelled using a single window of 12 m2. </p>
```

Comment:
@JavierArroyoBastida can you re-indent the documentation and revision history such that we avoid these long lines that cannot be read?

Reply:
done! https://github.com/open-ideas/IDEAS/pull/1146/commits/d3f7061a27ef07557fdab1cc7e1c632e18363858